dev_setup.sh

```diff
@@ -152,9 +152,8 @@ else
         fi
     fi
 
-    if ! [ "$has_mimic" == "" ] ; then
-        echo "Mimic is installed. Press 'y' to rebuild mimic, any other key to skip."
-        read -n1 build_mimic
+    if [ "$has_mimic" != "" ]; then
+        build_mimic="n"
```

Comment:
Should this be reversed since `build_mimic` is `n` by default? So it builds mimic if not present?

```bash
    if [ "$has_mimic" == "" ]; then
        build_mimic="y"
    fi

Reply:
Oh, yes, you're absolutely right!